supported_platforms.md

```diff
@@ -0,0 +1,51 @@
+# Microsoft Azure IoT SDKs for .NET
+
+This SDK is tested nightly on a mix of .NET frameworks on both Windows 10 and on Ubuntu 1604. For additional details
```

Comment:
I'd call this ".NET implementations", rather than .NET frameworks, since .netcoreapp isn't .NET framework (the whole naming thin